aip/general/0132.md

```diff
@@ -20,9 +20,9 @@ parameters), and return a list of responses matching that input.
 
 ## Guidance
 
-APIs **should** generally provide a `List` method for resources unless it is
-not valuable for users to do so. The purpose of the `List` method is to return
-data from a single, finite collection.
+APIs **must** provide a `List` method for resources unless the resource is a
+[singleton][]. The purpose of the `List` method is to return data from a single,
```

Comment:
Sorry, super late response here but easily fixed in a separate PR if necessary:

I think the verbiage "single, finite collection" isn't accurate when considering cross-collection lists AIP-159. Should that caveat be mentioned? @toumorokoshi